frr_python/frr_server.py

```diff
@@ -16,8 +17,16 @@
 from fleetspeak.src.inttesting.frr.proto.fleetspeak_frr.frr_pb2_grpc import MasterStub
 
 
-# TODO(Alexandr-TS): Make master server port passed in a flag.
-channel = grpc.insecure_channel('localhost:6059')
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string(
+    name="master_server_address",
+    default="localhost:6059",
+    help="Address of master server to forward clients' messages")
+
+
+FLAGS(["master_server_address"])
+channel = grpc.insecure_channel(FLAGS.master_server_address)
```

Comment:
It's not great that there is code in the top-level code (i.e. not in a function).

Could this be moved into main?

Even better would be to:
 - Turn Listener into a class
 - Give it a call operator (i.e. rename the current "def Listener(m,c)" into "def __call__(self, m, c)"
 - This way, an instance of this new class could be passed to service_client.Listen